CONTRIBUTING.md

```diff
@@ -25,6 +25,34 @@ Even better: You could submit a pull request with a fix / new feature!
    developers, or if you do not have permission to do that, you may request
    the second reviewer to merge it for you.
 
+## Release process
+
+To make it possible to automatically generate changelog, all tickets/issues must have a milestone and at least one label.
+
+A changelog can be generated using [`github-changelog-generator`][github-changelog-generator].<sup>(1)</sup>
+
+Our release versions follow [Semantic Versioning][semver].
+
+New release (and any related tags) are always created from the `master` branch
+
+To create a new release:
+
+1. Make sure all closed tickets and MRs have a label.
```

Comment:
For "normal" list format, there should not be any blank lines between the list items.

Reply:
I've always adhered to the [`lint-list-item-spacing` rule from Remark](https://github.com/remarkjs/remark-lint/tree/main/packages/remark-lint-list-item-spacing)

> According to the markdown-style-guide, if one or more list items in a list spans more than one line, the list is required to have blank lines between each item. And otherwise, there should not be blank lines between items.

So because of item 4 this list is a "loose" list. I could add a `.remarkrc` file + markdown linting... that should rule out any future incosistencies :thinking: .